src/common/components/Buttons/Button.tsx

```diff
@@ -31,6 +35,9 @@ const Button: React.FC<TButton> = ({
   const classes = cn(
     {
       'button inline-flex items-center px-2 h-10 justify-center rounded-2xl transition duration-300 focus:outline-none active:shadow-none focus:shadow-btnOutline max-w-full leading-none': true,
+      'w-full': fluid,
+      'w-60': !fluid && !width,
+      [`w-${width}`]: width,
```

Comment:
No, It is the hard code. We have to keep the tailwindcss. 
( otherwise postcss may remove needed classes during purging process for production build ) - For it, I added the part to implement the important.
![tailwindcss name]

Reply:
```
`w-${width}`
```
PostCSS plugin "PurgeCSS" is removing all unused classes from production build. And it can't see that class is used when you define you in such dynamic manner.

With tailwind or without it is hardcode anyway, so I suggested style, but here is a better approach:

https://tailwindcss.com/docs/just-in-time-mode#enabling-jit-mode
Please enable jit mode, and then you can use "w-[123px]" class, without defining it in tailwind config, much better approach.

This custom width case can be resolved without !important, I'm sincerity asking you to avoid !important in this case and in the future.